src/modules/element-info.js

```diff
@@ -30,10 +35,51 @@ class elementInfo {
     const commentsButton = document.createElement('button');
     commentsButton.classList.add('comments-button');
     commentsButton.textContent = 'Comments';
+    // code starts
     commentsButton.addEventListener('click', () => {
-      // displayCard(imgUrl, title, itemID, summary);
-    });
+      const modal = document.querySelector('.modal');
+      const popup = document.createElement('div');
+      popup.classList.add('popup');
+
+      const closeBtn = document.createElement('button');
+      closeBtn.innerHTML = '&#x3A7';
+      closeBtn.classList.add('close-btn');
+
+      const popupImg = image.cloneNode(true);
+      popup.classList.add('popup-img');
+
+      const comments = document.createElement('div');
+      getComments();
+      comments.textContent = 'Comments()';
+
+      const formDiv = document.createElement('div');
+      formDiv.classList.add('form-div');
+      const h3 = document.createElement('h3');
+      h3.textContent = 'Add a comment';
 
+      const form = document.createElement('form');
+      form.classList.add('comment-form');
+      const name = document.createElement('input');
+      const commentI = document.createElement('textarea');
+      const commentBtn = document.createElement('button');
+      commentBtn.textContent = 'Comment';
+      commentBtn.addEventListener('click', async () => {
+        await postComments(id, name.value, commentI.value);
+      });
+      form.append(name, commentI, commentBtn);
+      formDiv.append(h3, form);
+
+      popup.append(itemTitle.cloneNode(true), closeBtn, popupImg, comments, formDiv);
+
+      modal.style.display = 'block';
+      modal.appendChild(popup);
+
+      closeBtn.addEventListener('click', () => {
+        popup.remove();
+        modal.style.display = 'none';
+      });
+    });
+    // code ends
```

Comment:
Kindly create a new function using the selected code to have a better code design and follow the ES6 patterns :+1:

Reply:
Hi Romina. 
Thanks for the feedback.
 Refactored it. Created a new function called modal which then I use ES6 patterns to import